client/src/controllers/ProgressController.ts

```diff
@@ -57,6 +66,14 @@ export class ProgressController extends Controller<HTMLButtonElement> {
     document.addEventListener(
       'DOMContentLoaded',
       () => {
+        window.cancelSpinner = () => {
+          const attr = `data-${identifier}-loading-value`;
+
+          document.querySelectorAll(`[${attr}="true"]`).forEach((element) => {
```

Comment:
```suggestion
          document.querySelectorAll(`[${attr}~="true"]`).forEach((element) => {
```

Should change this selector to ensure that we find any elements with additional data-controller values.